packages/client/lib/util/parse.ts

```diff
@@ -234,54 +253,39 @@ async function parseGethParams(json: any) {
     [Hardfork.MuirGlacier]: 'muirGlacierBlock',
     [Hardfork.Berlin]: 'berlinBlock',
     [Hardfork.London]: 'londonBlock',
-    [Hardfork.PreMerge]: 'mergeForkBlock',
+    [Hardfork.MergeForkBlock]: 'mergeForkBlock',
   }
   params.hardforks = Object.values(Hardfork)
     .map((name) => ({
```

Comment:
Can we define the type here as `name: Hardfork` to make Typescript happy?

Reply:
What do you mean to make typescript happy?

Reply:
this is fine on my side, not seeing any issues or errors